lnwallet/wallet.go

```diff
@@ -186,6 +185,13 @@ type addCounterPartySigsMsg struct {
 	// version of the commitment transaction.
 	theirCommitmentSig []byte
 
+	// This channel is used for the wallet to pass the completed channel
+	// back into the Funding Manager at the final stage in the funding
```

Comment:
Hmm, I don't think we need to duplicate this variable in two locations: the `ChannelReservation` struct already has the `chanOpen` attribute which can be re-used here in place of both occurrences.